protected/views/patient/_fuzzy_date.php

```diff
@@ -18,46 +18,53 @@
  */
 ?>
 <?php
+/**
+ * @var $form FormLayout
+ * @var $label string
+ * @var $empty bool
+ */
 $label = @$label ?: 'Date';
 $labelColumns = @$form ? $form->columns('label') : 'large-3 column';
 $fieldColumns = @$form ? $form->columns('field') : 'large-9 column end';
 if (isset($date)) {
     list($sel_year, $sel_month, $sel_day) = explode('-', $date);
-} else {
+} elseif (!isset($empty) || !$empty) {
     $sel_day = $sel_month = null;
     $sel_year = date('Y');
+} else {
+    $sel_day = $sel_month = $sel_year = null;
```

Comment:
If users change the year from 'none' to 'January', can they still set it back to 'none'?